backend/docs/buildingsV1.yml

```diff
@@ -68,8 +78,9 @@ paths:
                 items:
                   $ref: "#/components/schemas/Building"
         "404":
-          description: Invalid
+          description: Building not found.
```

Comment:
So, honestly, we shouldn't return anything as a "404" for these catch-all endpoints.